torch/fx/pytree.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, Callable, List, Tuple, Type
```

Comment:
Why do these changes need to be made in this diff?

Reply:
If this is unrelated, I can remove this file change.

This file only simulates the behavior of `torch.fx._pytree` using the new `torch.utils.pytree`.

- `torch.utils.pytree` <-> `torch.utils._pytree`
- `torch.fx.pytree` <-> `torch.fx._pytree`

Reply:
I understand that, but my understanding was that the plan was to isolate this PR as "only adding dependencies", and then we can incrementally port all uses of pytrees over in subsequent PRs.

Reply:
Get it.